daffodil-lib/src/main/resources/org/apache/daffodil/xsd/DFDLGeneralFormatDefinition.dfdl.xsd

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Minor, but I think this name is excessively long. The "Definition" suffix adds nothing. Can this file just be DFDLGeneralFormat.dfdl.xsd.

Reply:
Sure thing